lib/simulator-xcode-6.js

```diff
@@ -617,6 +617,25 @@ class SimulatorXcode6 extends EventEmitter {
     return await settings.updateLocationSettings(this, bundleId, authorized);
   }
 
+  /**
+   * Enable/Disable reduce motion.
+   *
+   * @param {string} bundleId - The application ID to update.
+   * @param {boolean} reduceMotion - Whether or not to enable it.
+   */
+  async reduceMotion (bundleID, reduceMotion = true) {
+    if (await this.isRunning()) {
```

Comment:
I would not do daemon killing here, since it will an expected side effect

Reply:
@mykola-mokhnach the reduce motion change wont be available if we dont reboot the simulator. Let change that kill for reboot.